GENERATE_11ty_STATIC_SITE.md

```diff
@@ -0,0 +1,141 @@
+# Generating a static site with 11ty
+
+11ty is a very flexible static site generator, with loads of different templating and layout options. Below are some instructions to get you started but check the [11ty docs](https://www.11ty.dev/docs/getting-started/) for further customisation.
+
+## To create the 11ty starter
+1. Create a file name `.eleventy.js` at the root of your project
+1. Start with some basic config:
+    ```
+        const GIGSBOAT_INPUT_DIR = 'gigsboat-eleventy-starter'
+
+        module.exports = function(eleventyConfig) {
+            const markdownIt = require('markdown-it')
+            const options = { html: true }
+
+            eleventyConfig.setLibrary('md', markdownIt(options).use(require('markdown-it-anchor'), { permalink: false }))
+
+            eleventyConfig.addPassthroughCopy(`${GIGSBOAT_INPUT_DIR}/styles.css`)
+
+            return {
+                dir: { input: GIGSBOAT_INPUT_DIR }
+            }
+        }
+    ```
+1. Create a directory `gigsboat-eleventy-starter` where your 11ty starter files will live
+1. In the `gigsboat-eleventy-starter` directory create a file `index.json` and declare your layout file name
+`{ "layout": "gigsboat-layout" }`
+1. Create a file `` and add some basic css
```

Comment:
@estherixz @lirantal is there a file name missing on this line?

Reply:
this should be the `gigsboat-layout.njk` file I think...? @estherixz can you take a look?

Reply:
oops! 🤦🏽‍♀️ 😳  it's `styles.css`, will add it asap!

Reply:
@estherixz no worries, I added it in this commit https://github.com/gigsboat/cli/commit/f959823ef45434be8fee8a0ec3ca0c21433f0a7c

Thanks for replying!